gossip/ethapi_backend.go

```diff
@@ -289,7 +289,10 @@ func (b *EthAPIBackend) GetReceiptsByNumber(ctx context.Context, number rpc.Bloc
 	}
 
 	block := b.state.GetBlock(common.Hash{}, uint64(number))
-	receipts := b.svc.store.evm.GetReceipts(idx.Block(number), b.signer, block.Hash, block.Transactions)
+	time := uint64(block.Time.Unix())
+	baseFee := block.BaseFee
+	blobGasPrice := new(big.Int) // TODO: fix a price for this
```

Comment:
We have a range of lose ends regarding the blob pricing.

It would be great to have some systematic way to locate those when we work towards upgrading to Cancun. All of those will have to be addressed.